hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncTool.java

```diff
@@ -227,6 +227,9 @@ public static void main(String[] args) {
     }
     FileSystem fs = FSUtils.getFs(cfg.basePath, new Configuration());
     HiveConf hiveConf = new HiveConf();
+    if (cfg.hiveMetaStoreUri != null) {
+      hiveConf.set("hive.metastore.uris", cfg.hiveMetaStoreUri);
```

Comment:
So, IMO, let us introduce this config option when we do need it.

Reply:
When synchronizing to the remote hive, if this configuration is not added, the remote hive metastore will not be able to read the partition-related information

Reply:
Specific exception information can see this. https://issues.apache.org/jira/browse/HUDI-1528